images/linux/scripts/installers/java-tools.sh

```diff
@@ -29,11 +29,14 @@ echo "JAVA_HOME=/usr/lib/jvm/zulu-${DEFAULT_JDK_VERSION}-azure-amd64" | tee -a /
 echo "JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8" | tee -a /etc/environment
 
 # Install Ant
-apt-fast install -y --no-install-recommends ant ant-optional
-echo "ANT_HOME=/usr/share/ant" | tee -a /etc/environment
+curl -sL https://downloads.apache.org/ant/binaries/apache-ant-1.10.7-bin.zip -o ant.zip
```

Comment:
could you please clarify what is the purpose of replacing`apt install` to installation by direct link? We are trying to avoid adding something that will require manual maintenance in future. 

And the same question about JMeter installatiob, there are apt packages for it for both Ubuntu versions https://packages.ubuntu.com/search?keywords=jmeter&searchon=names&suite=all&section=all

Is there any specific reason to not use apt?

Reply:
Hello @AlenaSviridenko ,
Ubuntu packages are far from being up to date as of JMeter.
See https://askubuntu.com/questions/690983/how-can-i-update-jmeter-in-ubuntu

Latest version of jmeter can be found here:
https://jmeter.apache.org/download_jmeter.cgi#binaries
Would it help if we provided an additional stable name (without version suffix) for bundle ?

Regards

Reply:
Hi @pmouawad, yes, link with non-hardcoded version would be very convenient fro us, thank you!

Reply:
Hey @pmouawad! Are there any updates on the link with non-hardcoded version?

Reply:
Hello @AlenaSviridenko ,
There is no agreement met on this in the project team.
http://mail-archives.apache.org/mod_mbox/jmeter-dev/202005.mbox/%3CCAH9fUpYuMkA6AXEn%2Bc97Knvc_uYnbdPZqx3ruoJaQ%3D9kU8SwnQ%40mail.gmail.com%3E

Regards

Reply:
yeah, got it, thanks for the update

Reply:
Hey @pmouawad,
I saw in your emails that creation of latest release asset can bring more issues than help. 
I am not pushing you here, but just an idea: we could use GitHub API to get the latest release version from Jmeter GitHub releases, for example, https://api.github.com/repos/SeleniumHQ/selenium/releases/latest, but to do that latest release need to be marked as `Latest`. And then we could download the Jmeter binaries using this version.

anyway, that is not necessary, I think it is possible to use tags list https://api.github.com/repos/apache/jmeter/git/refs/tags but it just not so convenient as releases.

Reply:
@AlenaSviridenko , 
the releases in github are only source releases so it wouldn't work unfortunately.

Regards

Reply:
Hello @AlenaSviridenko,
Thanks for your suggestion, I have shared your advice on dev mailing list with following, can you confirm I am right regarding what needs to be done on our side ?

> Hello,
Alena suggest the option to mark last release as latest:

    https://github.com/actions/virtual-environments/pull/791

Anyway we currently don't distribute binaries on github to have this option work.

So to make it work , we would need to do it and use Github API in build:
1) To push additional assets (zip and tgz binaries):

    https://developer.github.com/v3/repos/releases/#edit-a-release-asset

2)  To mark latest:

    https://developer.github.com/v3/repos/releases/#edit-a-release


Thanks for your help

Reply:
it is not necessary to distribute binaries on GitHub, all we need is an ability to get the latest released version number. So there is no need in item 1 from your list. Just marking the latest release with `latest` tag should be enough to make it work